app/app-controllers/searchBarController.js

```diff
@@ -41,36 +42,40 @@ function(
 
     // You must manually call this for the search bar to be visible
     SearchBarController.prototype.addToLayout = function() {
-        this.layout.addTopView(this.viewPlugin, {height: 52});
+        this.layout.addTopView(this.viewPlugin);
         this.hide({animated: false}); // don't show it by default
     };
 
     SearchBarController.prototype._registerEvents = function() {
         var self = this;
-
         // Users of SearchBarController can also hook this event
         // to instruct the desired view to act on the search being performed
         // The search terms are expected to be in a string found in
         // params['searchTerms']
-        this.viewPlugin.on('search:submitted', function(params) {
-            self.hide({animated: true});
+        this.searchBar.on('search:submitted', function(params) {
+            self.hide();
         });
 
-        this.viewPlugin.on('search:cancelled', function(params) {
-            self.hide({animated: true});
+        this.searchBar.on('search:cancelled', function(params) {
+            self.hide();
         });
     };
 
     SearchBarController.prototype.show = function(options) {
+        options = options || (Astro.isRunningInIOSApp() ? {animated: true} : {animated: false});
```

Comment:
@guisers you can find examples of supplying default parameters here:
https://github.com/mobify/astro/blob/develop/js/src/plugins/navigationPlugin.js#L412

Out of curiosity though, is there a particular reason why we don't want show animations for android? (might be worth commenting here)